docs/_documentations/installoncloud.md

```diff
@@ -138,18 +138,13 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-## Adding additional rules to support Codewind ODO extension
-The Codewind ODO extension needs to add additional rules for accessing OpenShift resources:
-1. In your home directory, run the following command to clone the Codewind ODO extension repository:
-```
-git clone https://github.com/eclipse/codewind-odo-extension
-```
-2. Login to your OpenShift/OKD cluster
-3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules:
-```
-kubectl apply -f codewind-odoclusterrole.yaml
-kubectl apply -f codewind-odoclusterrolebinding.yaml
-```
+# Codewind OpenShift Do (odo) extension 
+See the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension) for the extension to Codewind that provides support for odo projects.
```

Comment:
Should be `OpenShift porjects` instead of `odo projects` since odo doesn't provide templates (not like Appsody).

Reply:
Update `Codewind OpenShift Do (odo) extension` to  `Codewind OpenShift Do (ODO) extension`

Reply:
Also `Codewind OpenShift Do (odo) extension` has the `<h1>` tag, so we should put it at the end to ensure it has the same level with `Installing Codewind on Eclipse Che` (another `<h1>` tag). I am thinking of we may need to create a new OpenShift section for 0.6.0 then put all information of ODO extension there to make it more clear.